app/controllers/api/v1/sessions_controller.rb

```diff
@@ -8,6 +9,15 @@ class SessionsController < Devise::SessionsController
       # POST /resource/sign_in
       def create
         if params[:type] == 'facebook'
+          user = obtain_facebook_user(params[:fb_access_token])
+          render json: { error: 'Not Authorized' }, status: :forbidden and return if user.nil?
```

Comment:
SessionsController doesn't extend from ApiController, so i couldn't do it. Is there a way to do it anyway?